data/human/news.txt

```diff
@@ -1990,6 +2111,79 @@ news "conspiracy theorist"
 
 
 
+news "player's crew member"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	to show
+		"flagship crew" > 40
+	name
+		word
+			"Crew member"
+	message
+		word
+			`"`
+		word
+			"I'm serving on board the '<ship>,' but I've never met the captain. Can you imagine that?"
+			"Hey captain, good to see you here. I'm looking forward to the next trip (and my 100 credits a day, of course)."
+			"You look like a poor excuse for a captain. I'm serving onboard the '<ship>' and I'm sure that my captain is far better dressed than you."
+		word
+			`"`
+
+
+
+news "sticker on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Sticker on ship"
+	message
+		word
+			"I brake for asteroid snails."
+			"Student pilot."
+			"Babe onboard."
+			"Baby onboard."
+			"There is no place like [insert homeplanet here]."
+			"This is not a bumper sticker, this is a rust retardant patch."
+			"Air leak seal, do not remove."
+
+
+
+news "label on ship"
+	location
+		government "Republic" "Free Worlds" "Neutral" "Independent" "Syndicate"
+	name
+		word
+			"Label on ship"
+			"Warning label on ship"
+	message
+		word
+			"Angles morts - Blind spots from cockpit."
+			"Stay clear of ramscoop intake."
+			"Stay out of thruster exhaust stream."
+			"Stay away from thruster outlets during takeoff."
+			"Door opens outwards."
+			"Remove red ribbons before flight."
+			"Nuclear reactor compartment: NEA certification is required for servicing this part of the ship."
+			"Compartiment du reacteur nucleaire: certification AEN requise pour l'entretien de cette partie du navire."
```

Comment:
I don't think the NEA will exist 1000 years into the future.

Reply:
Nobody said that the NEA 1000 years into the future is the same organization as the NEA we know these days.

Reply:
And I'm pretty sure that the organization will not be called "Nuclear Safety Agency", because other people will want to use the abbreviation NSA.

Reply:
Regardless I don't think we should be making references to modern day organizations in Endless Sky; it feels anachronistic and out of place. Could you change this to some sort of ES equivalent? 'Department of Republic Energy' or something like that?

Reply:
I think it also works without any agency name, so I just removed the agency name. (That's probably better than making up some agency by myself.)